### PR TITLE
Revert pkg bumps causing tests to fail on macs with sonoma

### PR DESCRIPTION
**Pull request by THenry14**
## Introduced changes

- Revert bumped dependencies causing cast tests to fail on macs with apple sillicon and newest updates installed

## Checklist

- [X] Linked relevant issue
- [X] Updated relevant documentation
- [X] Added relevant tests
- [X] Performed self-review of the code
- [X] Added changes to `CHANGELOG.md`

### CODE REVIEW

**Review by piotmag769 — APPROVED** [review]
🐦

File: crates/cast/Cargo.toml [hunk 1]
```diff
@@ -23,6 +23,8 @@ toml = "0.8.6"
 rpassword = "7.2.0"
 promptly = "0.3.1"
 scarb-artifacts = { path = "../scarb-artifacts" }
+# Freeze reqwest dependency, because newer versions cause cast tests to fail on macs
```
Comment (DelevoXDG) [inline crates/cast/Cargo.toml:26]:
```suggestion
# Freeze request dependency, because newer versions cause cast tests to fail on macs
```
😄

Comment (THenry14) [reply crates/cast/Cargo.toml:26]: obraz

Comment (DelevoXDG) [reply crates/cast/Cargo.toml:26]: Sorry, my bad

Comment (THenry14) [reply crates/cast/Cargo.toml:26]: no problem :D

**Review by cptartur — APPROVED** [review]

**Pull request merged**